my_view_page.php

```diff
@@ -151,7 +151,13 @@
 
 <?php if( $t_timeline_view_threshold_access ) { ?>
 <div class="col-md-5 col-xs-12">
-	<?php include( $g_core_path . 'timeline_inc.php' ); ?>
+	<?php
+		# Build a simple filter that gets all bugs for current project
+		$inc_timeline_filter = array();
+		$inc_timeline_filter[FILTER_PROPERTY_HIDE_STATUS] = array( META_FILTER_NONE );
+		$inc_timeline_filter = filter_ensure_valid_filter( $inc_timeline_filter );
```

Comment:
The approach i am using is to call filter_ensure_valid_filter:
- After creating a filter array, so it fills and formats all needed key fields.
- After modifying a filter from user input
- After loading a filter array from database, as it may fix minor format updates.

So, code that is modifiying a filter is responsible for validation.
The apis that use filters expect a valid structure, and shouldn't be needed to call  filter_ensure_valid_filter many times.